formidable/validators.py

```diff
@@ -68,7 +67,7 @@ class GTValidator(FormidableValidator, validators.BaseValidator):
 
     type = 'GT'
 
-    message = _("Ensure this field is greater than %(limit_value)s")
```

Comment:
For now these translations are in the formidable crowdin project, in a .po separated from the ui (see image)
If we remove the translation here, where should the translation happen now?
![screenshot from 2018-05-28 09-29-27](https://user-images.githubusercontent.com/26469767/40602704-d8735b9e-6259-11e8-9baa-00cd348090d1.png)

Reply:
the true question is: are these validators used anywhere?

I've grepped the django-formidable code, and they didn't appear anywhere. And I've grepped in our private-business application, and it's only used in one test, and just to make sure that the validator is correct.
I lack some time to investigate further, but for me, this looks like dead code. I think we can keep the module here and make it deprecated when we're more sure on this case.

Reply:
not that it's urgent, but... up?